src/config.ts

```diff
@@ -3,7 +3,7 @@ import * as Vec2 from 'lib/vec2'
 export const config = {
   scaling: window.devicePixelRatio,
   originalFrameInterval: 20,
-  pixelsPerMeter: (diagonal: number) => 18 * diagonal / 2000,
+  pixelsPerMeter: (diagonal: number) => diagonal / 250,
```

Comment:
Explain?

Reply:
If you think it is only of historical interest (personally I am not so sure, as we want to maintain the compatibility and it usually helps to understand the origin of any magic number), it might make sense to capture the context in the commit message instead of the code.